.github/workflows/docker_crossbuild_publish.yml

```diff
@@ -0,0 +1,64 @@
+name: Cross-Compile Docker Build and Push
+
+on:
```

Comment:
This is the correct event. But for the sake of testing this action could you please temporarily set: 

```yaml
on:
  pull_request:
    types:
      - opened
      - synchronize
```
So that this action runs on every commit to this branch(PR).

Reply:
Done.